Actions fails

With the following error:

```
  npm ERR! The `npm ci` command can only install with an existing package-lock.json or
  npm ERR! npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or
  npm ERR! later to generate a package-lock.json file, then try again.
  
  npm ERR! A complete log of this run can be found in:
  npm ERR!     /root/.npm/_logs/2020-10-25T19_44_30_633Z-debug.log
```

Looking at the `Dockerfile` I guess it's because there is no nodejs version constraint?

--- a/Dockerfile
+++ b/Dockerfile
@@ -8,7 +8,7 @@
 #
 # You can specify a version:
 # FROM node:10-slim
-FROM node:alpine
+FROM node:14-alpine3.12
 
 # Copy the package.json and package-lock.json
 COPY package*.json ./

